### PR TITLE
Implement 'complete' status icon for tab headers

### PR DESCRIPTION
**Pull request by ddzoan**
* this only implements complete icon for transmission. actual status component coming soon

### CODE REVIEW

File: tabs/TabHeader.jsx [hunk 1]
```diff
@@ -103,15 +116,16 @@ const styles = StyleSheet.create({
   tab: {
     color: latitudeColors.grey60,
     // $FlowUpgradeFixMe(0.110.1 -> 0.111.1)
-    ...include(padding.v.m),
+    paddingTop: whitespaceSizeConstants.m,
+    paddingBottom: whitespaceSizeConstants.m,
```
Comment (Cytancy) [inline tabs/TabHeader.jsx]: Are we moving away from the `padding.v.m`?

**Comment by lkartes-flexport** [comment]
Have you guys looked at https://github.com/flexport/latitude/pull/120 ? It's similar but allows you to choose any icon

File: tabs/TabHeader.jsx (hunk 1 above)
Comment (ddzoan) [reply tabs/TabHeader.jsx]: https://flexport.slack.com/archives/C5802T6A2/p1576021090080900

File: tabs/TabHeader.jsx [hunk 2]
```diff
@@ -92,6 +98,13 @@ const TabComponent = (props: {
       onClick={handleClick}
       role="button"
     >
+      {status === "complete" ? <Icon
```
Comment (notandrewkaye) [inline tabs/TabHeader.jsx]: can you add a TODO or a comment that explains that this is a temporary fix until https://flexport.atlassian.net/browse/LDS-463 gets implemented

**Pull request merged**